config.libsonnet

```diff
@@ -3,6 +3,22 @@
     // Selectors are inserted between {} in Prometheus queries.
     cephExporterSelector: 'job="rook-ceph-mgr"',
 
+    // Number of Ceph Managers which are reporting metrics
+    cephMgrCount: 3,
```

Comment:
Are we always expecting 3 MGR?

Reply:
So the alert checks if `x` number of replicas are running.

So `cephMgrCount` is a config option, where users set how many `cephMgrs` they are running. It is a *config* option, so users are expect to change it.

So should we change the default? If yes, what should the number be? Or drop alert entirely?

Reply:
From ceph team I heard that one of the mons ceph-mgr runs?
@leseb does Rook provide and option to decide the no of manager pods?

Reply:
If we want this config to be there. Lets have it as optional config. We can comment it out and if during the actual deployment the user can uncomment it. @shtripat @devopsjonas what do you think ?

Reply:
Certainly it should not be default configuration I feel.

Reply:
Maybe, instead we should just do `sum(up{%(cephExporterSelector)s}) < %(cephMgrCount)d` and set it to `1` by default. So that becomes `sum(up{job="ceph-mgr"}) < 1`, which means if we are not scraping metrics we would get alerted. This would work for everyone. WDYT?

Reply:
I've changed the default to be 1 and alert condition to fire than it's less thant that.